### ISSUE TITLE
Long opening theequalezer releases dv mp4 files 

### ISSUE THREAD

**Issue by galactic76** [justdan96/tsMuxer#733]
I count and its now 120 seconds for opening theequalizer releases dv mp4 files
version 380ff48 and in version e346384

I remember that you ask for mention wich version will be faster for opening that files to determine why is that way goes...

And its about the same time when counter of persentage stays on zero - also about 2 minutes

Its exactly the same time like in previous versions

BUT 380ff48 mux to TS with 30fps properties when show it on the screen of a player m9702, and no subtitles showing

I think its a some sort of a bug..... 

e346384 do the right fps and shows subtitles, but itshows 120 sec 100% and then start muxing and at the end shows 99,9% but file in fine

the middle version - 5231f3c hovewer significantly slower - to open a file in takes 230 sec instead on 120 sec

So, what version just fix 100% at the beginnign?

That all i need

And theolderst version seems the best

**Comment by jcdr428** [comment]
@galactic76 do you import the mp4 from a NAS or server, or is it on your PC ?

**Comment by galactic76** [comment]
From NAS server
And I found that 3 month older releases from that releaser opens at once, like files from all other releases.
I ask releaser what ру changed - he says nothing...

**Comment by jcdr428** [comment]
@galactic76 can you please identify the latest release opening "at once" without delay. It will help me find the changes that have led to this.

**Comment by galactic76** [comment]
No, I mean 3 month older releases from THAT releaser were opens at once in any of your version
Its not the matter of your releases. Is the matter of his releases. 
the last one that opens at once from him is a swan song and abizou, and older
Suwenney todd and showing up - already with big delay,,,

**Comment by jcdr428** [comment]
@galactic76 can you please try this [tsMuxer.exe](https://github.com/justdan96/tsMuxer/suites/16293069179/artifacts/930307245) (for win64) with your usual tsMuxerGUI.exe and report, thanks.

**Comment by galactic76** [comment]
No its even not open any file

**Comment by jcdr428** [comment]
Ok, small adjustment... Please try https://github.com/justdan96/tsMuxer/suites/16399542998/artifacts/936958047 or tomorrow's release.

**Comment by jcdr428** [comment]
@galactic76 can we close the issue ?

**Comment by galactic76** [comment]
No.
I try 051d59e
on film
The.Black.Phone.2021.2160p.UHDRemux.HDR.DV-TheEqualizer.mp4

it opens quick 
then error while muxing at 0%-
moov atom not found

**Comment by jcdr428** [comment]
@galactic76 there is a double thread mechanism in tsMuxer which I am not too familiar with: it is the reason why the commit is working on my PC in Debug mode, and not in Release mode or on your PC.
As a quick hack, I have introduced a 1-second sleep after the file seek to ensure that no bytes are read before the end of the file seek. Not the best method, but this should do the trick until somebody more knowladgeable in c++ looks at this (@xavery or @justdan96 maybe) ?

Please try tomorrow's release.

**Comment by galactic76** [comment]
it seems 2539d07 works fine!

**Comment by galactic76** [comment]
there is 2 more films that do not opens at all
Golda.2023.2160p.WEB-DL.Hybrid.HDR.DV-TheEqualizer.mp4
Reptile.2023.2160p.WEB-DL.Hybrid.HDR.DV-TheEqualizer.mp4

**Comment by jcdr428** [comment]
@galactic76 this seems to be another issue: the releases pre 17-Sept (before the commits on the current issue) outputed the same result.
Please close this issue if the long opening delay issue is confirmed fixed, and open a separate issue for the above mp4.

**Comment by galactic76** [comment]
solved thanks

**Comment by jcdr428** [comment]
@galactic76 I had to reverse on this commit as it creates unforeseen issues. Until this issue is fully solved, I reopen the issue.

**Comment by galactic76** [comment]
New 2 films that no ts muxer opens - 

Awareness.2023.2160p.WEB-DL.Hybrid.HDR.DV-TheEqualizer.mp4
The.Burial.2023.2160p.WEB-DL.Hybrid.HDR.DV-TheEqualizer.mp4

If it helps - clever FFMPEG says **Interlaced BFF detected**
